docs/source/technical_tutorials/search/searching_the_stac_catalog.ipynb

```diff
@@ -7,7 +7,11 @@
    "source": [
```

Comment:
<p>Output doesn't show here (it did work when I downloaded the notebook, just not sure why it's not popping up here)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/308/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I didn't run this simplified example cell, was just going to leave it as is. Should I run it?</p>

Reply:
I don't think it's necessary to run it if you wanted to leave it as is!